status-im/status-react
# Upgrade `react-native-camera-kit` library to resolve camera issues in Sign In screen

fixes #16226 #16063

### Summary

This PR upgrades the `react-native-camera-kit` library to `v13.0.0` to fix the standalone camera issues in our existing version (`v8.0.4`), especially in Android.

This new version ensures the standalone camera covers the whole screen in Android. And also helps us to prevent the background image from passing through in Sign In and causing a distortion effect with blur at the bottom of the screen in Android.

### Platforms

- Android
- iOS

### Areas impacted

Places where the standalone camera is used:

- Sign In/Pairing
- Adding a contact by scanning the QR code

### Steps to test

- Open Status
- Navigate to `Sign In`
- Check if the camera is covering the whole screen

status: ready
